client/extensions/woocommerce/app/promotions/promotion-form-conditions-card.js

```diff
@@ -0,0 +1,268 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import DatePicker from 'components/date-picker';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormInputCheckbox from 'components/forms/form-checkbox';
+import PriceInput from 'woocommerce/components/price-input';
+
+function editPromotionCoupon( siteId, promotion, data, editPromotion ) {
+	const coupon = promotion.coupon || {};
+	const newCoupon = { ...coupon, ...data };
+	editPromotion( siteId, promotion, { ...promotion, coupon: newCoupon } );
+}
+
+function editPromotionProduct( siteId, promotion, data, editPromotion ) {
+	const product = promotion.product || {};
+	const newProduct = { ...product, ...data };
+	editPromotion( siteId, promotion, { ...promotion, product: newProduct } );
+}
+
+const minMaxSpend = {
+	name: 'minMaxSpend',
+	getText: ( translate ) => translate( 'Minimum / Maximum spend' ),
```

Comment:
Just curious - what's the use case for a maximum spend?

Reply:
Lots of discounts have a maximum spend. I got a 10% off card for Lowes and it had a maximum of up to $250, limiting their exposure to $25.